GeosCore/photolysis_mod.F90

```diff
@@ -150,15 +150,16 @@ SUBROUTINE INIT_PHOTOLYSIS( Input_Opt, State_Chm, State_Diag, RC )
     ENDIF
 
     ! Compute the required wavelengths in the LUT to calculate requested AOD
-    IF ( .not. Input_Opt%DryRun ) THEN
+    IF ( notDryRun ) THEN
        IF (Input_Opt%amIRoot) WRITE(6,*) 'Wavelength optics read successfully'
        CALL CALC_AOD( Input_Opt, State_Chm, RC )
     ENDIF
 
     !--------------------------------------------------------------------
-    ! Exit if photolysis disabled (zero J-values)
+    ! Exit if photolysis disabled (zero J-values) or if dry-run
     !--------------------------------------------------------------------
     IF ( .NOT. Input_Opt%Do_Photolysis ) RETURN
+    IF ( Input_Opt%DryRun              ) RETURN
```

Comment:
You could put this before the block for computing AOD and then remove the if notdryrun bracket around the CALC_AOD call. I think this would be easier to understand since less if blocks.

Reply:
Now done in commit 631a8962.

Reply:
Looks good!